runners/spark/pom.xml

```diff
@@ -59,40 +59,40 @@
       <id>local-runnable-on-service-tests</id>
       <activation><activeByDefault>false</activeByDefault></activation>
       <build>
-        <pluginManagement>
-          <plugins>
-            <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-surefire-plugin</artifactId>
-              <executions>
-                <execution>
-                  <id>runnable-on-service-tests</id>
-                  <configuration>
-                    <groups>org.apache.beam.sdk.testing.RunnableOnService</groups>
-                    <parallel>none</parallel>
-                    <failIfNoTests>true</failIfNoTests>
-                    <dependenciesToScan>
-                      <dependency>org.apache.beam:java-sdk-all</dependency>
-                    </dependenciesToScan>
-                    <excludes>
-                      org.apache.beam.sdk.io.BoundedReadFromUnboundedSourceTest
-                    </excludes>
-                    <systemPropertyVariables>
-                      <beamTestPipelineOptions>
-                        [
-                          "--runner=TestSparkRunner",
-                          "--streaming=false"
-                        ]
-                      </beamTestPipelineOptions>
-                      <beam.spark.test.reuseSparkContext>true</beam.spark.test.reuseSparkContext>
-                      <spark.ui.enabled>false</spark.ui.enabled>
-                    </systemPropertyVariables>
-                  </configuration>
-                </execution>
-              </executions>
-            </plugin>
-          </plugins>
-        </pluginManagement>
+        <plugins>
```

Comment:
Well, why use `pluginManagement` to begin with ? it doesn't "manage" any sub-modules, and I haven't seen this in other runners' pom.